src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  background-color: black;
+}
+
+.box {
+  width: 328px;
+  margin: 20px 20px 40px;
+  padding: 10px;
+  border: 1px solid #e75551;
+}
+
+.blue_padding {
+  background-color: #308dc0;
+}
+
+.yellow_padding {
+  background-color: #ffa500;
+  padding: 10px;
+  border: 1px solid #e75551;
+}
+
+.text_block {
+  background-color: #ddd;
+  padding: 10px;
+  border: 1px solid #e75551;
+}
```

Comment:
1. quite a strange name, how can you have colored padding?
2. some of the styles are written several times, how about creating a general class and assigning all those styles to it instead of writing them several times?